### PR TITLE
Added GraphicsMetrics

### PR DESCRIPTION
**Pull request by Shqrdx**
Remake of https://github.com/mono/MonoGame/pull/3773

### CODE REVIEW

File: MonoGame.Framework/Graphics/GraphicsMetrics.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace Microsoft.Xna.Framework.Graphics
+{
+    /// <summary>
+    /// A snapshot of rendering statistics from <see cref="GraphicsDevice.Metrics"/> to be used for runtime debugging and profiling.
+    /// The metrics are reset every frame after draw within <see cref="GraphicsDevice.Present"/>.
```
Comment (tomspilman) [inline MonoGame.Framework/Graphics/GraphicsMetrics.cs]: Can you move this...
```
The metrics are reset every frame after draw within <see cref="GraphicsDevice.Present"/>. 
```
... to the docs for `GraphicsDevice.Metrics`.

How you have this now it implies that I every instance of `GraphicsMetrics` somehow magically resets on `GraphicsDevice.Present()` which actually only `GraphicsDevice.Metrics` resets like this.

**Comment by Shqrdx** [comment]
Done. Im correct ?

**Comment by tomspilman** [comment]
Yeah...  I think we're good now.

**Comment by tomspilman** [comment]
Ok merging!

**Pull request merged**